examples/03_connectivity/plot_gm_sphere_demo.py

```diff
@@ -0,0 +1,79 @@
+"""
```

Comment:
Sorry sorry it 's my rebase mess: I did a git push -f and you have all my local files.
I am removing them